lms/envs/common.py

```diff
@@ -600,7 +600,7 @@ def _make_mako_template_dirs(settings):
 
 
 CONTEXT_PROCESSORS = [
-    'lms.djangoapps.homepage.custom_context_processors.notifications_configs',
+    'lms.djangoapps.homepage.custom_context_processors.notifications_context',
```

Comment:
Ideally, we should not change code/logic in a unit test story, especially when QA is not considered. Minor changes can have unintended consequences.

Reply:
Understandable. The change was made at the suggestion of @zeshanarifios.

I think this change won't have any effect, as it is just a name change, but I myself don't have any concept related to context processors, so I can't say much. What do you think @zeshanarifios? Whatever you two decide among yourselves, or if some other reviewer contributes, I'll update accordingly.

Reply:
I agree with @zahid-arbisoft that ideally it should not be changed. 
I suggested because as it is just a name improvement regarding context processor, which isn't something that ll have unintended consequences untill we miss the usage in project.
Context processors are used only in `CONTEXT_PROCESSORS` settings and test cases. 

But still If zahid bhai feels it is unsafe, It is good to revert this.

Reply:
If it does not have any effect than its fine

Reply:
@ahmed-faraz46 would you like to shed some light here?

Reply:
I searched for its occurrences and found at just two places both are changed so I do agree with @zeshanarifios that in this particular case we can skip testing but I'd suggest (for this PR and for all future unit tests PRs) to not bind these decisions with either testing is required or not because it could be a possible subjective thing, each reviewer can have a different point of view on that, like in this case Zeeshan and I are agreed that testing is not required but someone else can say process is process, We've changed the code we need to get it tested. Which will waste a lot of time in discussions and back and forth. So here is what I am suggesting

While writing unit tests
1.  Make very necessary (it's not possible to write unit test without this change) changes as mentioned by @zahid-arbisoft 
2.  Above also includes the removal of duplication in tests and code, like defining constants and using them at both places in tests and code. And it should be responsibility of developer to verify it on local and other environments.

But I'd suggest to avoid renaming of a method regardless of the fact if testing is required or not. So in this particular case I'd suggest to revert the change.
Also @harris-irfan @zeshanarifios @zahid-arbisoft anything you want to add or remove in the above mentioned list? how about creating a guideline and sharing with the team to make sure we all are on the same page.

Reply:
I agree. Hence, reverting the name change.